config/src/main/java/com/typesafe/config/impl/SimpleConfig.java

```diff
@@ -884,7 +902,48 @@ public static long parseBytes(String input, ConfigOrigin originForException,
                 return result.longValue();
             else
                 throw new ConfigException.BadValue(originForException, pathForException,
-                        "size-in-bytes value is out of range for a 64-bit long: '" + input + "'");
+                    "size-in-bytes value is out of range for a 64-bit long: '" + input + "'");
```

Comment:
this is maybe what ConfigMemorySize.toBytes should throw for out of range

Reply:
exactly, thanks, fixed